src/cljs/rems/administration/components.cljs

```diff
@@ -199,6 +199,16 @@
 (defn inline-info-field [text value & [opts]]
   [info-field text value (merge {:inline? true} opts)])
 
+(defn localized-info-field
```

Comment:
in `rems.administration.form` seems it has been done manually by looping through languages. yes, looks like this could be a first step into unifying some fields logic :)